src/test/java/rx/SingleTest.java

```diff
@@ -38,10 +38,13 @@
 import rx.functions.Action1;
 import rx.functions.Func1;
 import rx.functions.Func2;
+import rx.observables.BlockingSingle;
 import rx.observers.TestSubscriber;
 import rx.schedulers.Schedulers;
 import rx.subscriptions.Subscriptions;
 
+import static org.junit.Assert.*;
```

Comment:
Wildcard imports are fine as they reduce the likelihood of PR collision. Saved us a lot in Observable.java